docs/status-reports/tbtc-2020-09-15-to-2020-10-13.adoc

```diff
@@ -0,0 +1,330 @@
+  bond loss.
+* Within a week, the tooling was completed to investigate the underlying key
+  shares. Though the result was not sufficient to recover the BTC, the
+  tooling forms the backbone of future key recovery needs. The intent is for
+  key recovery to be an unusual but well-supported part of client operations,
+  since it is the mechanism through which operators are generally able to
+  recover any value lost to liquidations.
+* Release 1.4.0 of the client was shipped this week shortly, adding additional
+  snapshotting of key material so that even if there is a different
+  possibility of overwriting key material, all key shares of all key
+  generations will have been captured in a separate directory without the
+  possibility of overwriting.
+
+== Other Failure Types
+
+A few other failure types were observed on mainnet, none of which resulted in
+liquidation, covered in summary form below:
+
+* **Depositors failing to fund deposits they have opened**. This results in a
+  loss to the depositors of the opening fee for the deposit. Signer bonds are
+  held for a short duration (3 hours) and can then be released by the
+  signers. A number of these have been observed on-chain.
+* **Depositors funding deposits incorrectly (by sending less than the required
+  amount of BTC to the signer group)**. None of these were observed on-chain
+  during the first month; however, in these cases the underlying BTC can be
+  recovered with some coordination, provided a cooperative signer group.
+  There is ongoing work to automate this coordination in the default case.
+* **Depositors failing to prove funding in time**. A BTC funding transaction
+  must be proven to have 6 confirmations within 3 hours of the deposit’s
+  having a public key, or the deposit can be closed by the signers. A proof
+  can fail to be submitted on time if a BTC transaction is mined too slowly
+  and the depositor does not increase their fee, or if the transaction has
+  its confirmations but the depositor never submits the proof to the chain.
+  One of these was observed during the first month; as with incorrect
+  funding, the underlying BTC is recoverable via coordination in these cases.
+  As with incorrect funding, there is ongoing work to automate this
+  coordination in the default case.
+* **Relay maintainer running out of gas**. The cross-chain nature of tBTC
+  requires an SPV relay, which enables Ethereum contracts to confirm that a
+  Bitcoin transaction exists on the Bitcoin chain and has been confirmed a
+  certain number of times. Without this component, funding and redemption
+  proofs cannot be submitted to tBTC, leading to the system's
+  deposit/redemption flow stalling out. Bitcoin can still flow, and any
+  unproven funds are still recoverable as with most other states the system
+  can enter, but obviously this is not ideal. The underlying causes for this
+  particular issue are still under investigation---a balance check
```

Comment:
`---` is rendered as just ---. Is it what we wanted to achieve?

Reply:
No, but it'll do.